src/main/java/org/gelecekbilimde/scienceplatform/post/model/request/CategoryCreateRequest.java

```diff
@@ -9,10 +12,27 @@
 @Setter
 @Builder
 public class CategoryCreateRequest {
-	@NotBlank
+
+	@NotBlank(message = "Category name cannot be blank")
+	@Size(max = 36, message = "Category name must be less than 36 characters")
```

Comment:
@aleyna-yildizli Özel bir mesaj yazmamaya çalışalımi buradaki mesajları bu şekilde özelleştirdiğimizde yazması zor olur ve uyuşmaz. Mümkün olduğunca spring'in kendi varsayılan mesajlarını kullanalım. 

Ek olarak kategori adı tek karakter olabilir mi? Örneğin `C` diye bir kategori olabilir mi? Eğer olamazsa buradaki minimum değeri de belirlememizde fayda var.

@Ciklamert Kategori adını veri tabanında 36 belirlemişiz. Sence yeterli olur mu? Eğer yeterli olmayacaksa 50, 100 gibi değerler verelim çünkü yarın öbür gün değiştirmesi biraz daha zor olabilir.

```suggestion
	@NotBlank
	@Size(max = 36)
```

Reply:
50 yeterli olur gibi
50 olarak belirtebiliriz. 36 çok arada bir sayı olmuş